pages/select_profile_page.py

```diff
@@ -47,8 +48,30 @@
 )
 contextual_data = dg.get_contextual_information()
 
+# Setting up the coloured scale bar chart for electricity and gas consumption
```

Comment:
basically we now create the colour scale bars outside of the page setup, because they are the same independently of the profile